components/html/modules/content-patterns/_typography.html

```diff
@@ -3,79 +3,79 @@
+					<h3 class="quote__content">"Archivo Narrow Bold 1.5em #2e2e2e Consectetur adipisicing elit. At illum esse quas sunt, reiciendis magnam itaque aut exercitationem et, voluptatum laboriosam expedita, quia officia ipsum nemo molestias alias sequi vero."</h3>
+					<p class="quote__source">- Archivo Narrow Bold 1.125em #98700d</p>
 				</aside>
 				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sed quidem similique, aspernatur illum tempora, nostrum pariatur eos consequuntur atque minus fugit deleniti illo reprehenderit architecto quis ipsam debitis asperiores temporibus!</p>
-				<aside class="quote quote-quotemark">
-					<h3 class="quote__content">"Archivo Narrow Bold 24px #2e2e2e Consectetur adipisicing elit. At illum esse quas sunt, reiciendis magnam itaque aut exercitationem et, voluptatum laboriosam expedita, quia officia ipsum nemo molestias alias sequi vero."</h3>
-					<span class="quote__source">- Archivo Narrow Bold 18px #98700d</span>
+				<aside class="quote quote-mark">
+					<h3 class="quote__content">"Archivo Narrow Bold 1.5em #2e2e2e Consectetur adipisicing elit. At illum esse quas sunt, reiciendis magnam itaque aut exercitationem et, voluptatum laboriosam expedita, quia officia ipsum nemo molestias alias sequi vero."</h3>
+					<span class="quote__source">- Archivo Narrow Bold 1.125em #98700d</span>
 				</aside>
 				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sed quidem similique, aspernatur illum tempora, nostrum pariatur eos consequuntur atque minus fugit deleniti illo reprehenderit architecto quis ipsam debitis asperiores temporibus!</p>
 				<hr />
 				<ul>
-					<li>Arial Regular 18px</li>
-					<li>Arial Regular 18px</li>
-					<li>Arial Regular 18px</li>
+					<li>Arial Regular</li>
+					<li>Arial Regular</li>
+					<li>Arial Regular</li>
 					<li>Nested<ul>
-							<li>Arial Regular 18px</li>
-							<li>Arial Regular 18px</li>
-							<li>Arial Regular 18px</li>
+							<li>Arial Regular</li>
+							<li>Arial Regular</li>
+							<li>Arial Regular</li>
 							<li>Nested<ul>
-								<li>Arial Regular 18px</li>
-								<li>Arial Regular 18px</li>
-								<li>Arial Regular 18px</li>
+								<li>Arial Regular</li>
+								<li>Arial Regular</li>
+								<li>Arial Regular</li>
 							</ul></li>
-							<li>Arial Regular 18px</li>
+							<li>Arial Regular</li>
 					</ul></li>
-					<li>Arial Regular 18px</li>
-					<li>Arial Regular 18px</li>
+					<li>Arial Regular</li>
+					<li>Arial Regular</li>
 				</ul>
 				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sed quidem similique, aspernatur illum tempora, nostrum pariatur eos consequuntur atque minus fugit deleniti illo reprehenderit architecto quis ipsam debitis asperiores temporibus!</p>
 				<ol>
-					<li>Arial Regular 18px</li>
-					<li>Arial Regular 18px</li>
-					<li>Arial Regular 18px</li>
+					<li>Arial Regular</li>
+					<li>Arial Regular</li>
+					<li>Arial Regular</li>
 					<li>Nested<ol>
```

Comment:
Voiceover reads the nested list-item elements differently than those that do not contain another ordered list. For example, it reads this element element "4" "period" "Nested", each being read separately. I'm not really sure how to handle this yet.